test/sslapitest.c

```diff
@@ -8000,6 +8000,9 @@ static int test_pluggable_group(int idx)
     if (!TEST_true(create_ssl_connection(serverssl, clientssl, SSL_ERROR_NONE)))
         goto end;
 
+    if (!TEST_true(!strcmp("XOR", SSL_groupid_to_name(serverssl, SSL_get_shared_group(serverssl, 0)))))
```

Comment:
Please wrap the line.

Reply:
Oops - will do - clearly didn't read https://www.openssl.org/policies/codingstyle.html in time :-(

Reply:
As noted above this only works (by accident) if `SSL_get_shared_group` happens to return a group with the `TLSEXT_nid_unknown` bit set. Otherwise it will return NULL.

I think we should have another test of this function that checks that a "built-in" group does the right thing.

Reply:
So added (a set of them, actually).